packages/orchestrator/internal/template/build/template_builder.go

```diff
@@ -296,7 +296,7 @@ func (b *TemplateBuilder) Build(ctx context.Context, template *TemplateConfig) (
 
 	// Cancel the start command context (it's running in the background anyway).
 	// If it has already finished, check the error.
-	readyCancel()
+	commandsCancel()
```

Comment:
there's a race condition, you need to make sure that the request for the start command was sent